robottelo/vm.py

```diff
@@ -52,25 +58,34 @@ def __init__(
             image_dir=None):
         self.cpu = cpu
         self.ram = ram
-        if distro is None or distro not in BASE_IMAGES:
-            self.distro = BASE_IMAGES[0]
-        else:
-            self.distro = distro
+        self.distro = BASE_IMAGES[-1] if distro is None else distro
```

Comment:
Sorry I was looking at the old list `BASE_IMAGES = ('rhel71', 'rhel70', 'rhel66', 'rhel65')` :)